pillar/journal-public.sls

```diff
@@ -4,7 +4,7 @@ journal:
     submit_url: https://submit.elifesciences.org/
     feature_xpub: false
     calls_to_action:
-      - path: ^(/$|/about$|/for-the-press$|/for-the-press/5cc21558/)
+      - path: ^(/$|/about$|/about/aims-scope|/for-the-press$|/for-the-press/5cc21558/)
```

Comment:
I took https://github.com/elifesciences/issues/issues/4769#issuecomment-469655965 to mean `/about$|/about/`